include/Chipset.h

```diff
@@ -207,6 +207,16 @@
 #ifdef OPENSHOCK_FW_BOARD_DFROBOTFIREBEETLE2ESP32E
 #define OPENSHOCK_BYPASSED_GPIO(pin) ((pin) == 2 || (pin) == 5)
 #endif
+#ifdef OPENSHOCK_FW_BOARD_NODEMCU32S
+#define OPENSHOCK_BYPASSED_GPIO(pin) ((pin) == 15 || (pin) == 2)
```

Comment:
Why are these pins bypassed?

Reply:
Pin 2 is for the built in LED and pin 15 was just because the other boards do it. You can remove pin 15 if you'd like :3

Reply:
It should be good now!